clang/docs/checkedc/Update-to-latest-LLVM-sources.md

```diff
@@ -23,13 +23,20 @@ The third step is to merge your changes back into your baseline and master branc
 First create remote to the mirrored GitHub repo that contains the updated sources
 for LLVM/Clang. Go to your LLVM/Clang repo and do:
 
-    git remote add mirror https://github.com/llvm-mirror/llvm-project
+    git remote add mirror https://github.com/llvm/llvm-project
 
 Then branch your baseline branch and merge changes into it:
 
     git checkout baseline
     git checkout -b updated-baseline
     git pull mirror master
+    git reset --hard <llvm-commit>
```

Comment:
Rather than starting the `updated-baseline` branch at `baseline` only to hard-reset it to a different commit, you can start it at the commit you want; the effect will be the same.
```
git fetch mirror
git checkout -b updated-baseline <llvm-commit>
```